TOOODO/Data/Sources/Repository/CategoryRepository.swift

```diff
@@ -15,8 +16,12 @@ public class CategoryRepositoryImpl: CategoryRepository {
         self.localDataSource = localDataSource
     }
     
-    public func getCategories() -> Array<Domain.Category> {
-        localDataSource.getCategories().map { $0.toDomain() }
+    public func getCategories() -> AnyPublisher<Array<Domain.Category>, Error> {
+        localDataSource.getCategories()
+            .map {
+                $0.map { $0.toDomain() }
+            }
+            .eraseToAnyPublisher()
```

Comment:
컴바인을 사용하면 이런 코드들이 조금 복잡해요 ㅠㅠ
만약 async/await였다면 아래처럼 될텐데요
```suggestion
        try await localDataSource.getCategories().toDomain()
```

Reply:
그래서 저는 Network, Data, Domain 레이어에서는 최대한 concurrency를 지향합니다